src/main/java/com/o19s/grandcentral/kubernetes/PodManager.java

```diff
@@ -343,7 +344,7 @@ else if (left.getLastRequest() < right.getLastRequest())
    * @throws IOException
    */
   private void refreshPods() throws IOException {
-    HttpGet podsGet = new HttpGet("https://" + k8sConfiguration.getMasterIp() + ":443/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
+    HttpGet podsGet = new HttpGet(k8sConfiguration.getProtocol() + "://" + k8sConfiguration.getMasterIp() + "/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
```

Comment:
I like having an optional protocol for that reason.  It's less to think about when you are configuring it.  In almost all cases (except test) we can assume it's https.